src/rules/utils/selectors/index.ts

```diff
@@ -6,3 +6,4 @@ export const store = `MethodDefinition[kind='constructor'] TSTypeReference[typeN
 const actionDispatch =
   'ExpressionStatement:has(CallExpression > MemberExpression:has(Identifier[name="dispatch"]):has(MemberExpression > Identifier[name="store"]))'
 export const multipleActionDispatch = `${actionDispatch} ~ ${actionDispatch}`
+export const dispatchInEffects = `ClassProperty > CallExpression:has(Identifier[name="createEffect"]) CallExpression > MemberExpression:has(Identifier[name="dispatch"]):has(MemberExpression > Identifier[name="store"])`
```

Comment:
Using `:has` has had some unexpected results to me in the past, I'm going to merge it for now it I might change this query later on. Or if you want to rewrite the query, be my guest 🙂